pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -516,7 +516,13 @@ func (p *MasqueradePodInterface) createBridge() error {
 }
 
 func (p *MasqueradePodInterface) createNatRules() error {
+	if Handler.UseIptables() {
+		return p.createNatRulesUsingIptables()
+	}
+	return p.createNatRulesUsingNftable()
```

Comment:
Naming is not consistent. createNatRulesUsingIptable**s** vs createNatRulesUsingNFtable. Just a nit if you find other problems, I don't think it's worth running all tests again.